shared/login/register/code-page/index.render.desktop.js

```diff
@@ -57,9 +57,9 @@ class CodePageRender extends Component<void, Props, void> {
         <div style={stylesQrContainer}>
           <div style={{...qr, ...stylesQr}} />
         </div>
-        <p style={{...globalStyles.flexBoxRow, alignItems: 'flex-end'}} onClick={() => this.props.setCodePageMode(codePageModeEnterText)}>
+        <p style={{...globalStyles.flexBoxRow, alignItems: 'flex-end'}} onClick={() => this.props.setCodePageMode(codePageModeShowText)}>
           <Icon style={{marginRight: 15}} type='icon-phone-text-code-32' />
-          <Text type='BodyPrimaryLink' onClick={() => this.props.setCodePageMode(codePageModeEnterText)}>Type text code instead</Text>
+          <Text type='BodyPrimaryLink' onClick={() => this.props.setCodePageMode(codePageModeEnterText)}>Show text code instead</Text>
```

Comment:
@MarcoPolo Thanks, this onClick turned out to be a noop (worked without the suggested change), so I just removed the inner onClick from the Text component altogether.